app/uk/gov/hmrc/gform/sharedmodel/SmartString.scala

```diff
@@ -41,6 +43,8 @@ case class SmartString(localised: LocalisedString, interpolations: List[Expr]) {
 
   def transform(fEn: String => String, fCy: String => String): SmartString =
     copy(localised = localised.transform(fEn, fCy))
+
+  def mapExpr(f: Expr => Expr): SmartString = this.modify(_.interpolations.each).using(_.mapExpr(f))
```

Comment:
It looks like `def mapExpr(f: Expr => Expr)` is ever called only on `Expr` only, why this (and others) then? Can it be removed?